### PR TITLE
Change copyright titlte

### PR DESCRIPTION
**Pull request by namho102**

### CODE REVIEW

**Comment by namho102** [comment]
refer #161

**Comment by namho102** [comment]
refer #160

**Comment by hynguyen2610** [comment]
It's look good for me, but I need more chars

**Comment by hynguyen2610** [comment]
OK, now it's good

**Comment by hynguyen2610** [comment]
Open

File: backend/src/const/constants.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /** Constants */
 export const UPVOTE_ACTION    =   1;
 export const DOWNVOTE_ACTION  =   -1;
-export const HISTORY_LIMIT    =   20;
```
Comment (hynguyen2610) [inline backend/src/const/constants.js]: Just a demo: This line is not good

File: backend/src/controllers/station.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ import user from '../routes/user';
 const MAX_SONG_UNREGISTED_USER_CAN_ADD = 3;
 const POINTS_FOR_FIRST_SONG = 2;
 const POINTS_FOR_NEXT_SONG = 1;
+const MINIMUM_DURATION = 60;
```
Comment (hynguyen2610) [inline backend/src/controllers/station.js]: This line is good to

**Pull request closed without merging**